src/FixedDataTable.js

```diff
@@ -243,6 +243,16 @@ var FixedDataTable = createReactClass({
     headerHeight: PropTypes.number.isRequired,
 
     /**
+     * Pixel height of fixedDataTableCellGroupLayout/cellGroupWrapper.
+     * Default is headerHeight and groupHeaderHeight.
+     * By being able to target this class specifically, we can do things
+     * like make some cells of a header smaller/bigger than those with
+     * the actual header height, with just a bit of additional css
+     * (override the overflow property of some other classes).
```

Comment:
Do we need lines 248-251? I think the first two lines of this comment are enough

Reply:
Removed

Reply:
There's some value in knowing the use case.  Could you add a third line like "This can be used with CSS to make a header cell span both the group & normal header row."

Reply:
Thanks, though I don't want to be misleading - if you have a group header present and decide to use the prop, the cells will cover the regular header, and the regular header will also spill out into the table (if you add the css to show the overflow). 

You basically would not want to use a group header if you use the prop, you would want to create a new table for the other header like we do in our codebase. 

I'm not sure how to communicate all this in a concise comment though. Maybe adding another sentence like: 

    ...
    * It's not recommended though to use this with the group header, as it
    * likely won't produce a desired styling. Instead, use another table as
    * a faux group header, and use the prop only in that header.

Reply:
Ah, thanks for the correction.  I like the direction of your sentence, but try to gear it towards describing precisely how you use it, rather than cases it doesn't support.

e.g. "Setting this to a value larger than height will cause the content to overflow the height.  This is useful when adding a 2nd table as the group header and vertically merging the 2 headers when no group is set.

Reply:
I agree, and thank you so much for the suggestion!

How is this? Is it still too much/not clear enough?

    /**
     * Pixel height of fixedDataTableCellGroupLayout/cellGroupWrapper.
     * Default is headerHeight and groupHeaderHeight. 
     * 
     * This can be used with CSS to make a header cell span both the group & normal header row.
     * Setting this to a value larger than height will cause the content to 
     * overflow the height. This is useful when adding a 2nd table as the group 
     * header and vertically merging the 2 headers when a column is not part 
     * of a group. 
     *
     * The CSS would involve changing both headers' cellGroupWrapper to have overflow-x: hidden
     * and pointer-events: none, while doing pointer-events: auto on cellGroup to 
     * reenable them on child els. For the group header table, you also need overflow: visible 
     * and a higher z-index for Layout/main, overflow-y: visible for CellLayout/main, and finally 
     * overflow: visible for cellGroup.
     */

Reply:
Ya that works for me.
You might see if you can condense the 3rd paragraph to bullet points, but if not it's good as it.

Reply:
Done, thank you! Much better.

    /**
     * Pixel height of fixedDataTableCellGroupLayout/cellGroupWrapper.
     * Default is headerHeight and groupHeaderHeight.
     *
     * This can be used with CSS to make a header cell span both the group & normal header row.
     * Setting this to a value larger than height will cause the content to
     * overflow the height. This is useful when adding a 2nd table as the group
     * header and vertically merging the 2 headers when a column is not part
     * of a group. Here are the necessary CSS changes:
     *
     * Both headers:
     *  - cellGroupWrapper needs overflow-x: hidden and pointer-events: none
     *  - cellGroup needs pointer-events: auto to reenable them on child els
     * Group header:
     *  - Layout/main needs overflow: visible and a higher z-index
     *  - CellLayout/main needs overflow-y: visible
     *  - cellGroup needs overflow: visible
     */